### ISSUE TITLE
NPE com.dynview.config.Configuration.getCommonConfig() from Compact Machines fabric-port mod

### ISSUE THREAD

**Issue by Shadowrs** [ldtteam/DynView#33]
Not sure which mod is causing this sorry
Please check out this [Issue](https://github.com/AlphaMode/CompactMachines-Fabric/issues/3)

**Comment by someaddons** [comment]
log?

**Comment by Shadowrs** [comment]
https://mclo.gs/H0G19GG

```java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'compactmachines'!
	at net.minecraft.class_310.md9b2c1a$lambda$invoke$0$4(class_310.java:10490)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.minecraft.class_310.invoke(class_310.java:10488)
	at net.minecraft.class_310.redirect$cca000$init(class_310.java:10475)
	at net.minecraft.class_310.<init>(class_310.java:452)
	at net.minecraft.client.main.Main.main(Main.java:197)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.polymc.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104)
	at org.polymc.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:175)
	at org.polymc.impl.OneSixLauncher.launch(OneSixLauncher.java:185)
	at org.polymc.EntryPoint.listen(EntryPoint.java:144)
	at org.polymc.EntryPoint.main(EntryPoint.java:74)
	Suppressed: java.lang.NoClassDefFoundError: Could not initialize class net.minecraft.class_3230
		at re.zarex.simplechunkloader.SimpleChunkLoader.<clinit>(SimpleChunkLoader.java:34)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:467)
		at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50)
		at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117)
		at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53)
		at net.minecraft.class_310.invoke(class_310.java:10486)
		... 15 more
Caused by: java.lang.ExceptionInInitializerError
	at dev.compactmods.machines.upgrade.ChunkloadUpgrade.<clinit>(ChunkloadUpgrade.java:18)
	at dev.compactmods.machines.upgrade.MachineRoomUpgrades.<clinit>(MachineRoomUpgrades.java:24)
	at dev.compactmods.machines.upgrade.graph.RoomUpgradeConnection.lambda$static$0(RoomUpgradeConnection.java:13)
	at com.mojang.serialization.codecs.RecordCodecBuilder.create(RecordCodecBuilder.java:72)
	at dev.compactmods.machines.upgrade.graph.RoomUpgradeConnection.<clinit>(RoomUpgradeConnection.java:13)
	at dev.compactmods.machines.graph.GraphEdgeType.<clinit>(GraphEdgeType.java:13)
	at dev.compactmods.machines.graph.CMGraphRegistration.lambda$static$6(CMGraphRegistration.java:31)
	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:183)
	at io.github.fabricators_of_create.porting_lib.util.RegistryObject.get(RegistryObject.java:44)
	at io.github.fabricators_of_create.porting_lib.util.LazyRegistrar.lambda$register$0(LazyRegistrar.java:46)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at io.github.fabricators_of_create.porting_lib.util.LazyRegistrar.register(LazyRegistrar.java:45)
	at dev.compactmods.machines.graph.CMGraphRegistration.init(CMGraphRegistration.java:39)
	at dev.compactmods.machines.CompactMachines.onInitialize(CompactMachines.java:62)
	at net.minecraft.class_310.invoke(class_310.java:10486)
	... 15 more
Caused by: java.lang.NullPointerException: Cannot invoke "com.dynview.config.Configuration.getCommonConfig()" because the return value of "com.dynview.DynView.getConfig()" is null
	at net.minecraft.class_3230.<clinit>(class_3230.java:22)
	... 30 more
```

**Comment by someaddons** [comment]
fixed

**Comment by ArrowM** [comment]
@someaddons Could you please backport this fix to the 1.18 release?

**Comment by ArrowM** [comment]
Just made PR #36 for it.

https://media.tenor.com/vCqpL7x4sPUAAAAC/thanos-fine-ill-do-it-myself.gif
